ci/cpu/upload.sh

```diff
@@ -29,45 +29,35 @@ fi
 
 gpuci_logger "Starting conda uploads"
 if [[ "$BUILD_LIBCUDF" == "1" && "$UPLOAD_LIBCUDF" == "1" ]]; then
-  export LIBCUDF_FILE=$(conda build --no-build-id --croot "${CONDA_BLD_DIR}" conda/recipes/libcudf --output)
-  test -e ${LIBCUDF_FILE}
-  echo "Upload libcudf"
-  echo ${LIBCUDF_FILE}
-  gpuci_retry anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-rapidsai} ${LABEL_OPTION} --skip-existing ${LIBCUDF_FILE} --no-progress
+  export LIBCUDF_FILES=$(conda build --no-build-id --croot "${CONDA_BLD_DIR}" conda/recipes/libcudf --output)
+  while read -r LIBCUDF_FILE; do
```

Comment:
Should you be looping over `LIBCUDF_FILES` to generate each `LIBCUDF_FILE`?

Reply:
Yeah agree.

AJ, thoughts on doing all of this in a single `anaconda upload` line?

Reply:
If you look at the bottom of the `while` loop, you'll see this line:

```sh
done <<< "${LIBCUDF_FILES}"
```

which uses `LIBCUDF_FILES` as the input. `LIBCUDF_FILE` is the variable name for the current iteration of the `while` loop. The best example of this I could find online is on this page under the `Using a here strings` section:

- https://www.cyberciti.biz/faq/unix-howto-read-line-by-line-from-file/

Reply:
> AJ, thoughts on doing all of this in a single anaconda upload line?

No objections here. What's the best way to accomplish that?

Reply:
> AJ, thoughts on doing all of this in a single anaconda upload line?

Sounds like an improvement to me! What's the best way to accomplish this?

Reply:
Ah I missed the here-string. I'm used to here-docs but didn't think to look at the bottom of the while loop, don't see this construct nearly as often. I agree that if you could do a single anaconda upload that would be great though.

Reply:
From looking at the docs here, it seems like we can simply pass multiple file names to the `anaconda upload` command. I'll test this locally and update the PR accordingly.

- https://docs.anaconda.com/anacondaorg/commandreference/#upload

Reply:
fixed in 04094b4